### ISSUE TITLE
Gtk-CRITICAL - assertion failures

### ISSUE THREAD

**Issue by john-davies** [john-davies/mapter#3]
Opening the editor on a cell in a completely blank row produces the following:

````
(mapter:6082): Gtk-CRITICAL **: gtk_text_buffer_set_text: assertion 'text != NULL' failed
(mapter:6082): Gtk-CRITICAL **: gtk_text_buffer_set_text: assertion 'text != NULL' failed
````
If any cell on that row has text then the assertions don't appear

Also get 

````
(mapter:6082): GLib-CRITICAL **: g_string_erase: assertion 'string != NULL' failed
(mapter:6082): GLib-CRITICAL **: g_string_overwrite_len: assertion 'string != NULL' failed
````
the first time that the text is saved.

Everything seems to work OK but these warnings should not appear

**Comment by john-davies** [comment]
Problem only happens with rows and columns added at the end. Saving the project causes a core dump

**Comment by john-davies** [comment]
Fixed in 0.3.1
